web-animations/timing-model/animation-effects/local-time.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>AnimationEffect local time tests</title>
+<link rel="help" href="http://w3c.github.io/web-animations/#local-time">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../testcommon.js"></script>
+<body>
+<script>
+'use strict';
+
+test(function(t) {
+  var anim = createDiv(t).animate({}, 10 * MS_PER_SEC);
+  for (var seconds of [-1, 0, 5, 10, 20]) {
+    anim.currentTime = seconds * MS_PER_SEC;
+    assert_equals(anim.effect.getComputedTiming().localTime, seconds * MS_PER_SEC);
+  }
+}, 'Local time is current time for AnimationEffects associated with an Animation.');
```

Comment:
Since AnimationEffect is not an item in the current spec interface, this statement is a little confusing. Should we use AnimationEffectReadOnly, or perhaps just talk about "animation effects"?